openapi/openapi.yaml

```diff
@@ -609,6 +612,9 @@ components:
         testId:
           description: A unique id for test script run
           type: string
+        baseImage:
```

Comment:
Do we need return this as part of the TestJobResult?

Reply:
@wanghaoran1988 we don't need to - but I felt it may be a useful field for a managed script user (if they were running multiple scripts with different images and comparing the results, let's say). I've removed it now - it's not strictly necessary.